src/main_menu.cpp

```diff
@@ -68,10 +69,17 @@ class demo_ui : public cataimgui::window
         void on_resized() override {
             init();
         };
+
+    private:
+        cataimgui::Paragraph stuff;
 };
 
 demo_ui::demo_ui() : cataimgui::window( _( "ImGui Demo Screen" ) )
 {
+    // stuff = cata_imgui::parse_colored_text( "Some long text that will wrap around nicely. </color><color_green><color_red>Some red text in the </color>middle.</color> Some long text that will <color_light_blue_yellow>wrap around nicely.", c_white );
```

Comment:
I expect this will be deleted. Otherwise, your color tags don't match.

Reply:
Then I recommend adding a comment along the lines of "mismatched colour tags work too:"

Otherwise, it just looks like a mistake.